config/webpack/webpack.config.js

```diff
@@ -14,6 +14,7 @@ const ThemeAggregator = require('../../scripts/aggregate-themes/theme-aggregator
 const getThemeWebpackPromise = require('./getThemeWebpackPromise');
 const ThemePlugin = require('./postcss/ThemePlugin');
 const getThemeConfig = require('./postcss/_getThemeConfig');
+const ThemeLinterPlugin = require('./theme-linter-plugin/ThemeLinterPlugin');
```

Comment:
Could you make the exports separate? Naming this stuff sucks.
```suggestion
const ThemeLinterPlugin, { ThemeVarCollectorPlugin } = require('./theme-linter-plugin/ThemeLinterPlugin');
```

Reply:
Would it make sense to for the Theme Linter Plugin to live next in the postcss/ dir with the ThemePlugin? these seem like they go hand-in-hand.